src/client.js

```diff
@@ -53,6 +64,7 @@ const makeClient = options => {
           method: 'POST',
           headers,
           body: JSON.stringify({query, variables: (variables || {})}),
+          agent: clientContext.httpsAgent,
```

Comment:
This property is only used by node-fetch, which is what underlies isomorph in CLI.